src/main/java/edu/colorado/cires/cmg/tracklinegen/BaseRowListener.java

```diff
@@ -20,6 +19,7 @@
 
 public class BaseRowListener<T extends DataRow> implements RowListener<T> {
 
+  private final long NmSplit;
   private final long msSplit;
```

Comment:
Minor nitpick: Can this be "nmSplit" to be consistent with Java variable name conventions?